app/views/components/_published-dates.html.erb

```diff
@@ -0,0 +1,30 @@
+<%
+  published ||= false
+  history ||= false
+  last_updated ||= false
+  link_to_history ||= false
+%>
+
+<div class="app-c-published-dates" <% if history%>id="history" data-module="toggle"<% end %>>
+  <% if published %>
+    Published <%= published %>
+  <% end %>
+  <% if last_updated %>
+    <br />Last updated <%= last_updated %>
```

Comment:
If there's no published date this will include an unneeded last updated date. Sadly this can happen on bulk imported specialist documents (like old AAIB reports) – on these documents we show when it was updated but suppress published date.

Reply:
Might be worth adding an example showing this too.